modules/distribution/src/assembly/bin.xml

```diff
@@ -35,69 +59,37 @@
-                <exclude>**/repository/components/plugins/wss4j_1.5.11.wso2v9.jar</exclude>
-                <exclude>**/repository/components/plugins/nimbus-jose-jwt_2.26.1.wso2v2.jar</exclude>
-                <exclude>**/repository/components/plugins/net.minidev.json-smart_1.2.0.jar</exclude>
-                <!--<exclude>**/repository/components/plugins/wss4j_1.5.11.wso2v10.jar</exclude>-->
-                <exclude>**/repository/components/plugins/rampart-trust_1.6.1.wso2v14.jar</exclude>
-                <exclude>**/repository/components/plugins/rampart-policy_1.6.1.wso2v14.jar</exclude>
-                <exclude>**/repository/components/plugins/rampart-core_1.6.1.wso2v14.jar</exclude>
-                <exclude>**/repository/components/plugins/commons-io_2.0.0.wso2v2.jar</exclude>
-                <exclude>**/repository/components/plugins/com.google.gson_2.1.0.jar</exclude>
-                <exclude>**/repository/components/plugins/com.google.gson_2.2.4.jar</exclude>
-                <!--<exclude>**/repository/components/plugins/org.wso2.carbon.captcha.mgt_4.2.0.jar</exclude>-->
-                <exclude>**/repository/components/plugins/guava_12.0.0.wso2v1.jar</exclude>
-                <exclude>**/repository/components/plugins/wadl-core_1.1.3.wso2v2.jar</exclude>
-                <!--<exclude>**/repository/components/plugins/waffle_1.4.0.wso2v1.jar</exclude>-->
-                <!--<exclude>**/repository/components/plugins/compass_2.0.1.wso2v2.jar</exclude>-->
-                <!--<exclude>**/repository/components/plugins/poi_3.9.0.wso2v1.jar</exclude>-->
-
-                <exclude>**/conf/thrift-authentication.xml</exclude>
-                <exclude>**/repository/components/features/org.wso2.carbon.apimgt.store_${apim.feature.version}/store/**</exclude>
-                <exclude>**/repository/components/features/org.wso2.carbon.apimgt.store_${apim.feature.version}/admin-dashboard/**</exclude>
-                <exclude>**/repository/components/features/org.wso2.carbon.apimgt.store_${apim.feature.version}/workflow-admin/**</exclude>
-                <exclude>**/repository/components/features/org.wso2.carbon.apimgt.publisher_${apim.feature.version}/publisher/**</exclude>
-                <exclude>**/repository/components/features/org.wso2.carbon.apimgt.core_${apim.feature.version}/libs/**</exclude>
-
-                <!--below dependency fails synapse startup-->
-                <exclude>**/repository/components/plugins/asm-all_4.1.0.wso2v1.jar</exclude>
-
-                <exclude>**/sso-idp-config.xml</exclude>
-                <exclude>**/application-authentication.xml</exclude>
-                <exclude>**/sso/jaggery.conf</exclude>
-                <exclude>**/sso/views/register_new_user.jag</exclude>
+                <exclude>**/lib/httpmime*</exclude>
+                <exclude>**/lib/encoder*</exclude>
+                <exclude>**/plugins/jaxb*</exclude>
+                <exclude>**/security/authenticators.xml</exclude>
+
```

Comment:
Already fixed in commit https://github.com/wso2-incubator/product-iot-server/commit/febdc36cba8b94c3ebcaba27947893ebb2865513 as per the PrabathA's comment.